package.json

```diff
@@ -0,0 +1,105 @@
+{
+  "name": "terra-framework",
+  "private": true,
+  "version": "0.1.0",
```

Comment:
Should this be v0.0.0?

Reply:
This is what terra-core is currently using. I'm not sure if it matters.

Reply:
Oh right. This was likely updated we did our v1 release at the end of Q2